biz.aQute.bndlib/src/aQute/bnd/build/api/OnWorkspace.java

```diff
@@ -0,0 +1,57 @@
+package aQute.bnd.build.api;
+
+import java.io.Closeable;
+import java.util.Collection;
+import java.util.function.Consumer;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+import aQute.bnd.build.Project;
+import aQute.bnd.build.Run;
+import aQute.bnd.build.Workspace;
+import aQute.bnd.service.RepositoryPlugin;
+
+/**
+ * The Workspace can notify interested parties when certain events happen. These
+ * events can be called multiple times since in an IDE, the workspace can change
+ * its properties, its projects, and its repositories.
+ * <p>
+ * Callbacks will always happen on another thread and will not be called after
+ * the close has returned.
+ */
+@ProviderType
+public interface OnWorkspace extends Closeable {
```

Comment:
Is there a reason to use `Closeable` instead of `AutoCloseable`?

Reply:
any reason?